modelon/impact/client/entities/workspace.py

```diff
@@ -254,7 +255,51 @@ def upload_result(
         )
         return ExternalResultUploadOperation(resp["data"]["id"], self._sal)
 
-    def download(self, options: Dict[str, Any], path: str) -> str:
+    def export(self, options: Dict[str, Any]):
+        """Exports the workspace as a binary compressed archive.
+        Returns an modelon.impact.client.operations.workspace
+        .WorkspaceExportOperation class object.
```

Comment:
Because the export and download end-points are so similar I think it would be good if there was some cross referencing in the docstrings of them. Like 'Similar to 'download' but gives more control for getting the workspace async' and 'Similar to 'export' but does the entire setup and download as in one go'. Not sure about best ways.